docs/controllers/services/annotations.md

```diff
@@ -34,6 +34,10 @@ Specifies which algorithm the Load Balancer should use. Options are `round_robin
 
 Specifies which stick session type the loadbalancer should use. Options are `none` or `cookies`.
 
+### service.beta.kubernetes.io/do-loadbalancer-sticky-sessions-cookie-name
+
+Specfies what cookie name to use for the DO loadbalancer sticky session. This annotation is required if `service.beta.kubernetes.io/do-loadbalancer-sticky-sessions-type` is set to `cookies`.
+
 ### service.beta.kubernetes.io/do-loadbalancer-sticky-sessions-cookie-ttl
 
 Specifies the TTL of cookies used for loadbalancer sticky sessions. This annotation is required if `service.beta.kubernetes.io/do-loadbalancer-sticky-sessions-type` is set.
```

Comment:
Would you mind appending

```
to `cookies`
```

to the last sentence in order to be consistent with your added paragraph above?

Reply:
@timoreimann I am not sure I understand this comment.

Reply:
Oh nvm I got it